product-tours/how-it-works/docs/Neptune-API-Tour.ipynb

```diff
@@ -44,31 +53,43 @@
   {
    "cell_type": "markdown",
    "metadata": {
-    "tags": [
-     "header",
-     "installation"
-    ]
+    "pycharm": {
+     "name": "#%% md\n"
+    }
    },
    "source": [
-    "## Before you start"
+    "## Before you start\n",
+    "\n",
+    "This notebook example lets you try out Neptune as an anonymous user, with zero setup.\n",
+    "\n",
+    "* If you are running the notebook on your local machine, you need to have [Python](https://www.python.org/downloads/) and [pip](https://pypi.org/project/pip/) installed.\n",
+    "* If you want to see the example recorded to your own workspace instead:\n",
+    "    * Create a Neptune account → [Take me to registration](https://neptune.ai/register)\n",
+    "    * Create a Neptune project that you will use for tracking metadata → [ Tell me more about projects](https://docs.neptune.ai/administration/projects)"
```

Comment:
```suggestion
    "    * Create a Neptune project that you will use for tracking metadata → [Tell me more about projects](https://docs.neptune.ai/administration/projects)"
```

@SiddhantSadangi If you are copying this `[ Tell me more about projects]` from somewhere, please make sure to remove the extra space after the bracket. It's appearing consistently.

Reply:
I think this is happening due to the black precommit hook.
It wont impact how the notebook is rendered

Reply:
Hmm, the change was introduced to the notebook template after you moved the arrow outside the link. That looks like a manual change

Reply:
Fixed